src/components/Counter.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+
+function Counter(props) {
+  return (
+    <div className="counterContainer">
+      <p>
```

Comment:
Use material's Typography component for displaying text and you can customise it easily.

Reply:
Okay